profiles/Amandeep.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/107784525/182681880-2fa3d360-76b2-4a96-8bb1-20978f31ef8f.png",
+  "issueId": 1278,
+  "name": "Amandeep Singh",
+  "username": "Amandeep404"
+}
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
This should also fix the Prettier (or Continuous Integration) issues.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable